features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -347,6 +372,26 @@ So we suggest ensuring that you are using the correct height and width dimension
 {% endtab %}
 {% endtabs %}
 
+#### Examples - Object aware cropping (cm-extract)
+
+Instead of specifying the `x`, `y`, `xc` or `xy` coordinates to focus on a particular area, you can also set `fo` value to an object from the [supported list](#supported-object-list), which will intelligently detect `xc`, `yc` values as the center of the provided object and extract the image with the given width and height.
```

Comment:
```suggestion
Instead of specifying the `x`, `y`, `xc`, or `xy` coordinates to focus on a particular area, you can also set `fo` value to an object from the [supported list](#supported-object-list), which will intelligently detect `xc`, `yc` values as the center of the provided object and extract the image with the given width and height.
```